### PR TITLE
7658 edit petitioner UI

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/views/ServiceIndicatorRadios.jsx [hunk 1]
```diff
@@ -28,6 +30,8 @@ export const ServiceIndicatorRadios = connect(
     const selectNone =
       bindObject.serviceIndicator === SERVICE_INDICATOR_TYPES.SI_NONE;
 
+    const showElectronic = isElectronicAvailable !== false;
```
Comment (rachelschneiderman) [inline web-client/src/views/ServiceIndicatorRadios.jsx:33]: Nit: Is there a reason to use a different variable other than `isElectronicAvailable`?

**Review by rachelschneiderman — APPROVED** [review]

File: web-client/src/views/ServiceIndicatorRadios.jsx (hunk 1 above)
Comment (matthopson) [reply web-client/src/views/ServiceIndicatorRadios.jsx:33]: I'm doing this so it defaults to true. If it's not EXPLICITLY `false` we should show electronic. Though, perhaps changing it to just "hideElectronic" solves that a little more easily.

Comment (rachelschneiderman) [reply web-client/src/views/ServiceIndicatorRadios.jsx:33]: Ah, gotcha. Thanks!

Comment (matthopson) [reply web-client/src/views/ServiceIndicatorRadios.jsx:33]: I think I'm going to refactor this. Probably in a follow up PR.

**Pull request merged**